Apps/WebClient/src/appsettings.json

```diff
@@ -65,8 +65,17 @@
             "Dependent": true
         },
         "HoursForDeletion": 720,
-        "MinPatientAge" : 12,
-        "MaxDependentAge": 12
+        "MinPatientAge": 12,
+        "MaxDependentAge": 12,
+        "OfflineMode": {
+            "StartDateTime": "2020/06/01 12:00:00PM",
```

Comment:
since this is app settings lets not have it here but instead in Development

Reply:
moved